pkg/replica/replica.go

```diff
@@ -62,6 +60,9 @@ type Replica struct {
 	revisionCounterDisabled bool
 
 	unmapMarkDiskChainRemoved bool
+
+	snapshotMaxCount int
+	SnapshotMaxSize  int64
```

Comment:
Why exported?

Reply:
I updated naming and didn't notice that. Thank you.